src/lib/InstantSearch.js

```diff
@@ -365,8 +365,8 @@ See: https://www.algolia.com/doc/guides/building-search-ui/widgets/create-your-o
           this._isSearchStalled = true;
           this._render(
             this.helper,
-            this.helper.lastResults,
-            this.helper.lastResults._state
+            this.helper && this.helper.lastResults,
+            this.helper.lastResults && this.helper.lastResults._state
```

Comment:
however the state maybe could be local if there's no last results? Something like 

```suggestion
            this.helper.lastResults ? this.helper.lastResults._state : this.helper.state
```

Reply:
Your suggestion produces this error when I try to navigate to another page.

![Screen Shot 2020-05-20 at 9 15 59 AM](https://user-images.githubusercontent.com/11767079/82450435-b44f3980-9a7a-11ea-9912-25529b954497.jpg)

Reply:
I think that error is related to the line above more than this one. Maybe we should find a way to create an empty SearchResults and use that to render when there's not already a lastResults. (results is null, not state). 

I'd love to try out what you have though, I have not seen those errors before, so I think something else is going on as well

Reply:
I'm working on the sandbox now.

Reply:
@Haroenv I haven't been able to reproduce this error in a sandbox. I setup the project to have the same exact implementation of Algolia / vue-instantsearch and I cannot surface this error. However I can assure you this error has happened thousands of times in our production application. How should I proceed to get this solved?

Reply:
@Haroenv jumping in because @ctbradley is on my team and the error is happening in our product.

We've seen 7,757 occurrences of this error in our application. Replicating in a sandbox doesn't seem to surface the problem, but we also can't seem to identify a difference that would allow us to change our production application to solve the problem. Any advice you could give up on next steps would be fantastic.